CondFormats/SiPixelObjects/src/SiPixelCalibConfiguration.cc

```diff
@@ -3,7 +3,7 @@
 #include <cstdlib>
 
 short SiPixelCalibConfiguration::vcalIndexForEvent(const uint32_t & eventnumber) const{
-  uint32_t relative_event = abs(eventnumber-1)%patternSize();
+  uint32_t relative_event = (eventnumber-1)%patternSize();
```

Comment:
yes, i just am not sure if that is the desired behavior. I'll let that get answered in the review process.